docs/.vuepress/components/Disqus.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div id="disqus_thread"></div>
+</template>
+<script>
+export default {
+  mounted() {
+    var disqus_config = function() {
```

Comment:
have you really used a `var` bro? XD

Reply:
This should be global for disqus

Reply:
🤦‍♂️

Reply:
global as in window global ?
Using var would not have `disqus_config` be available globally right 🤔 

Does it work as intended at the moment ?